### PR TITLE
[Expert] Speed increases

### PR DESCRIPTION
**Pull request by MuteTiefling**
Dramatically increases the speed for the natural altar's ore processing step. Also increases aura cost per tick. 

Make the IE Mixer actually worth using. Makes it aproximately 8x faster than a maxed out (256 rotation) Create Mixer

### CODE REVIEW

File: kubejs/server_scripts/enigmatica/kubejs/expert/unification/unify_materials.js [hunk 1]
```diff
@@ -282,8 +282,8 @@ onEvent('recipes', (event) => {
                 output: Ingredient.of(levigated_material).toJson(),
                 catalyst: Ingredient.of('naturesaura:crushing_catalyst').toJson(),
                 aura_type: 'naturesaura:overworld',
-                aura: 1500,
-                time: 10
+                aura: 300,
```
Comment (NielsPilgaard) [inline kubejs/server_scripts/enigmatica/kubejs/expert/unification/unify_materials.js:285]: I believe your intention was to increase this?

Comment (MuteTiefling) [reply kubejs/server_scripts/enigmatica/kubejs/expert/unification/unify_materials.js:285]: yep, it's doubled. was 150 per tick, now it's 300 per tick

Comment (MuteTiefling) [reply kubejs/server_scripts/enigmatica/kubejs/expert/unification/unify_materials.js:285]: er, wait where'd the time change go >.>

Comment (MuteTiefling) [reply kubejs/server_scripts/enigmatica/kubejs/expert/unification/unify_materials.js:285]: ok, no time is still 1, so 300 per tick :D

Comment (NielsPilgaard) [reply kubejs/server_scripts/enigmatica/kubejs/expert/unification/unify_materials.js:285]: The old value was 1500 per tick though, according to the diff above?

Comment (MuteTiefling) [reply kubejs/server_scripts/enigmatica/kubejs/expert/unification/unify_materials.js:285]: 1500 over ten. It's the total cost, not per tick cost

Comment (NielsPilgaard) [reply kubejs/server_scripts/enigmatica/kubejs/expert/unification/unify_materials.js:285]: Ahh okay, thanks 😅

**Pull request merged**